product_pricelist_revision/models/pricelist.py

```diff
@@ -8,7 +8,8 @@ class ProductPricelistItem(models.Model):
     _inherit = "product.pricelist.item"
     _rec_names_search = ["name"]  # Initialize here to be able to extend
 
-    name = fields.Char(store=True, compute_sudo=True)
+    name = fields.Char(store=True)
+    price = fields.Char(compute_sudo=True)
```

Comment:
Dont change type of price. It is not a char.

Reply:
Weirdly this has been running in production for years now. It was just a rebase to resolve a merge conflict. Think I'm just going to close it.